modules/aks/azure/values.tmpl.yaml

```diff
@@ -75,6 +75,9 @@ cert-manager:
       enabled: true
     acme:
       solvers:
+        - selector:
```

Comment:
I think the selector belongs to the same block than dns01.

Reply:
Fixed by d8291cc